### PR TITLE
Fix the culture parent chain for zh-Han[s|t]-*

### PR DESCRIPTION
**Pull request by tarekgh**
https://github.com/dotnet/runtime/pull/44620
https://github.com/dotnet/runtime/issues/44502
https://github.com/dotnet/runtime/pull/44607

### CODE REVIEW

**Comment by tarekgh** [comment]
@safern could you please have a look? this is the delta change we added to the servicing PR https://github.com/dotnet/runtime/pull/44620.

File: src/libraries/System.Globalization/tests/CultureInfo/CultureInfoParent.cs [hunk 1]
```diff
@@ -16,10 +16,20 @@ public class CultureInfoParent
...
-            CultureInfo culture = new CultureInfo(name);
-            Assert.Equal(new CultureInfo(expectedParentName), culture.Parent);
+            try
+            {
+                CultureInfo culture = new CultureInfo(name);
+                Assert.Equal(new CultureInfo(expectedParentName), culture.Parent);
+            }
+            catch (CultureNotFoundException)
+            {
+                // on downlevel Windows versions, some cultures are not supported e.g. zh-Hans-CN
```
Comment (safern) [inline src/libraries/System.Globalization/tests/CultureInfo/CultureInfoParent.cs:30]: Nit: Something to consider here, maybe assert we are running on a downlevel version of Windows?

**Review by safern — APPROVED** [review]

File: src/libraries/System.Globalization/tests/CultureInfo/CultureInfoParent.cs (hunk 1 above)
Comment (tarekgh) [reply src/libraries/System.Globalization/tests/CultureInfo/CultureInfoParent.cs:30]: I am not worried about that as if this happen with any of these cultures will show up in many other places in our tests.

Comment (safern) [reply src/libraries/System.Globalization/tests/CultureInfo/CultureInfoParent.cs:30]: Sounds good.

**Pull request merged**